src/pages/docs/octopus-rest-api/cli/octopus-release-create.mdx

```diff
@@ -46,6 +46,7 @@ Global Flags:
 $ octopus release create --project MyProject --channel Beta --version 1.2.3
 $ octopus release create -p MyProject -c Beta -v 1.2.3
 $ octopus release create -p MyProject -c default --package "utils:1.2.3" --package "utils:InstallOnly:5.6.7"
+$ octopus release create -p MyProject -c default --package "utils:1.2.3" --package "Run a Script:app_package_ref:1.1.1"
```

Comment:
I added this additional example to try highlight the StepName:PackageRefName and that package ref names will not include : but instead replace them with a `_`